forms-flow-idm/migration/utils.py

```diff
@@ -253,3 +253,161 @@ def add_user_to_group(access_token, user_id, group_id):
     headers = {"Authorization": f"Bearer {access_token}"}
     response = requests.put(url, headers=headers)
     response.raise_for_status()
+
+def role_changes_710():
```

Comment:
Wouldn't it make more sense to have this in the same `migrate-<version>.py` file?

Reply:
This function contains lists that are commonly used in both migrate-7.1.0.py and migrate-7.2.0.py, so it has been moved to the utils file for reuse.